### PR TITLE
Update msp.rst

### PR DESCRIPTION
**Pull request by chintanr97**
I learnt that Hyperledger Fabric does not provide custom OU support in profile section. But the documentation at https://hyperledger-fabric.readthedocs.io/en/master/msp.html#best-practices suggests in point number 2.1 that we can define policies which capture the custom OUs of the identity.

Signed-off-by: Chintan Rajvir <chintanr97@gmail.com>

#### Type of change

- Bug fix
- New feature
- Improvement (improvement to code, performance, etc)
- Test update
- Documentation update

### CODE REVIEW

File: docs/source/msp.rst
Comment (jyellick) [inline docs/source/msp.rst]: This isn't entirely accurate.  There are four different types of identity principals: https://github.com/hyperledger/fabric-protos/blob/3bc53193070ecef4edd4ccc361c728d818c98128/msp/msp_principal.proto#L41-L49

The two that are being conflated here, are OU based ones, and Role based ones.

When 'NodeOUs' are enabled for an MSP, it ties four OUs to four roles.  These roles are Peer, Orderer, Client, and Admin, and typically, we use an OU of the same name.

You may additionally specify identities in policies directly by OU (for instance, maybe you want to create an OU for each region, or business unit of your clients).  Since these identities are more free-form, creating sane tooling to deal with this is challenging, and doesn't exist yet.

So, you may absolutely specify identity roles defined by NodeOUs, but these are roles, not OUs (even if the OU is of the same name).  There is no way to specify identity OUs via configtx.yaml, though the underlying support exists in the MSP.

**Review by jyellick — CHANGES_REQUESTED** [review]

File: docs/source/msp.rst
Comment (chintanr97) [reply docs/source/msp.rst]: _You may additionally specify identities in policies directly by OU (for instance, maybe you want to create an OU for each region, or business unit of your clients). Since these identities are more free-form, creating sane tooling to deal with this is challenging, and doesn't exist yet._

Yes so because it is not possible to create a policy like "OR('Org1MSP.department1')", where _department1_ is an OU (even after enabling NodeOUs and adding the specific OU to config.yaml); it becomes very necessary to resolve the confusion with respect to fololowing document line:

_Policies can then be defined to capture members of a specific OU, and these policies may constitute the read/write policies of a channel or endorsement policies of a chaincode._ 

The above line should specifically mention that after enabling NodeOUs, it is only the default OUs that can be referenced in access policies. This is what I learnt from my doubt in StackOverflow:

https://stackoverflow.com/questions/60256416/adding-custom-node-ous-with-hyperledger-fabric

As Gari suggests, it is important to be specific in the particular segment of the code.

Comment (chintanr97) [reply docs/source/msp.rst]: You may additionally specify identities in policies directly by OU (for instance, maybe you want to create an OU for each region, or business unit of your clients). Since these identities are more free-form, creating sane tooling to deal with this is challenging, and doesn't exist yet.

Yes so because it is not possible to create a policy like "OR('Org1MSP.department1')", where department1 is an OU (even after enabling NodeOUs and adding the specific OU to config.yaml); it becomes very necessary to resolve the confusion with respect to fololowing document line:

Policies can then be defined to capture members of a specific OU, and these policies may constitute the read/write policies of a channel or endorsement policies of a chaincode.

The above line should specifically mention that after enabling NodeOUs, it is only the default OUs that can be referenced in access policies. This is what I learnt from my doubt in StackOverflow:

https://stackoverflow.com/questions/60256416/adding-custom-node-ous-with-hyperledger-fabric

As Gari suggests, it is important to be specific in the particular segment of the code.

Comment (chintanr97) [reply docs/source/msp.rst]: Please let me know if you want me to edit the doc in a more specific way with your views on accepting the PR!

Comment (pamandrejko) [reply docs/source/msp.rst]: @chintanr97 I think if you change:
 be defined to capture members of a specific ``OU`` (should be one of: peer, admin,..
to
 be defined to capture members of a specific ``role`` (should be one of: peer, admin, 

it would be clearer.

There maybe other places in the documentation that would need to be corrected as well.  I can look into that.

Comment (chintanr97) [reply docs/source/msp.rst]: @pamandrejko I have pushed another commit to PR making changes as suggested by you. Please let me know if any further changes are needed before merge.

**Review by pamandrejko — APPROVED** [review]
LGTM

File: docs/source/msp.rst
Comment (chintanr97) [reply docs/source/msp.rst]: @jyellick Can you please check for the same for 2nd approval!

**Comment by chintanr97** [comment]
> LGTM

@pamandrejko Need one more approval!

**Comment by pamandrejko** [comment]
@btl5037 Since we only require a single +1 here, what is preventing this from being merged? Jason's comments are marked resolved.

**Comment by chintanr97** [comment]
> @btl5037 Since we only require a single +1 here, what is preventing this from being merged? Jason's comments are marked resolved.

@pamandrejko It says:

"1 review requesting changes and 1 approving review by reviewers with write access." I did not understand this!

**Review by jyellick — APPROVED** [review]

**Pull request merged**
